SU-SWS/stanford_profile
# D8CORE-5729 People term pages: display only child terms groupings

# READY FOR REVIEW

# Summary
- If a person is tagged with multiple types/groups, we don't want to display types/groups that are not relevant to the current taxonomy term we are viewing.
- This actually also fixes the ticket D8CORE-5657 which needed the headers of the lists adjusted.

# Need Review By (Date)
- 4/28

# Urgency
- medium

# Steps to Test
1. checkout this branch
2. import configs
3. configure the taxonomy terms so that there are at least 2 parents with some child tags under those.
4. create a person and tag with multiple child terms from different parents
5. view the parent taxonomy term page
6. verify the child from the other parent isn't displayed.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
